src/main/java/Uranus/Tasks/Task.java

```diff
@@ -0,0 +1,52 @@
+package Uranus.Tasks;
+
+public class Task {
+    protected String description;
+    protected String commandInput;
+    protected boolean isDone;
+    private String tag;
+
+    public Task(String description) {
+        this.commandInput = description;
+        setDescription(description);
+        this.isDone = false;
+        this.tag = " "; // for no-tag
```

Comment:
Be very careful on the usage of an empty String and a String that contains a space character!